### PR TITLE
Update Verizon

### PR DESCRIPTION
**Pull request by villasenor**
- Consolidates Verizon and Verizon Wireless (Login domain is Verizon.com)
- Adds the available text and email options (more detail is available once you are logged in, but the linked FAQ describes the 2fa)

### CODE REVIEW

File: _data/utilities.yml [hunk 1]
```diff
@@ -506,18 +506,11 @@ websites:
...
-
-  - name: Verizon Wireless
-    url: https://www.verizonwireless.com/
-    twitter: VZWSupport
-    img: verizon.png
-    regions:
-      - us
+    tfa:
+      - sms
+      - email
```
Comment (phallobst) [inline _data/utilities.yml:512]: The FAQ page also mentions "My Verizon app notifications".
```suggestion
      - email
      - proprietary
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/utilities.yml (hunk 1 above)
Comment (villasenor) [reply _data/utilities.yml:512]: lgtm

**Review by phallobst — APPROVED** [review]

File: _data/utilities.yml [hunk 2]
```diff
@@ -506,18 +506,12 @@ websites:
...
+      - proprietary
+    doc: https://www.verizon.com/support/keeping-your-account-safe-faqs/
```
Comment (ApCoder123) [inline _data/utilities.yml:514]:
```suggestion
    doc: https://www.verizon.com/support/keeping-your-account-safe-faqs/
    regions:
      - us
```

**Review by ApCoder123 — DISMISSED** [review]

File: _data/utilities.yml [hunk 3]
```diff
@@ -506,16 +506,12 @@ websites:
...
-    regions:
-      - us
-
-  - name: Verizon Wireless
-    url: https://www.verizonwireless.com/
-    twitter: VZWSupport
     img: verizon.png
+    tfa:
+      - sms
+      - email
```
Comment (ApCoder123) [inline _data/utilities.yml:512]: Where about is email 2FA mentioned on the page? I can only see references to the "My Verizon app" and SMS

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**